site/2017/10/24/odm-1-2-0-and-1-1-7.rst

```diff
@@ -0,0 +1,81 @@
+Doctrine MongoDB ODM 1.2.0 and 1.1.7 Released
+=============================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.2.0 <https://github.com/doctrine/mongodb-odm/releases/tag/1.2.0>`_ and
+`1.1.7 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.7>`_.
+
+MongoDB ODM 1.1.7
+-----------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/1.1.x/CHANGELOG-1.1.md#117-2017-10-23>`_.
+A full list of issues and pull requests included in this release may be found in the
+`1.1.7 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.1.7>`_.
+
+What is new in 1.2.0?
+---------------------
+
+Doctrine MongoDB ODM 1.2.0 comes with the following new features:
+
+- The ``readOnly`` mapping option allows you to map immutable documents.
+- With ``slaveOkay`` being deprecated, you can now specify a ``readPreference``
+  for your documents. This ``readPreference`` will automatically be applied to
+  all queries run on the collection.
+- With ``dbRef`` only offering limited support in aggregation pipeline queries,
+  there's a new reference storage strategy called ``ref`` that stores references
+  as objects, allowing you to use it with discriminators and aggregation
```

Comment:
"as objects (without "$"-prefixed field names)" should make this immediately clear to readers.